src/components/LandingPage/LandingPage.css

```
@@ -0,0 +1,101 @@
+#landing-page-container {
+    display: flex;
+    justify-content: center;
+    max-width: 100%;
+}
+
+#info-section {
+    display: flex;
+    flex-direction: column;
+    max-width: 40%;
+    flex: 1;
+    margin-top: 5%;
+    margin-right: 3%;
+}
+
+#title {
+    font-weight: 700px;
+    color: #000000;
+    font-size: 54px;
+    font-style: normal;
+    letter-spacing: 0.035em;
+    line-height: 56px;
+    display: flex;
+    text-align: left;
+}
+
+#lab-description {
+    font-weight: 400;
+    font-size: 20px;
+    line-height: 24px;
+    letter-spacing: 0.04em;
+    display: flex;
+    text-align: left;
+    margin-top: 15px;
+}
+
+.btn_ubc_home {
```

Just wondering why this particular name uses underscores instead of dashes? As in not lie: 

```suggestion
.btn-ubc-home {
```

Also wondering if there is any specific reason for primarily using `id`'s instead of `className`'s in your element tags? 🤔